pytripgui/main_window_qt_vc/main_window_cont.py

```diff
@@ -45,6 +49,59 @@ def _initialize(self):
         self.view.about_callback = self.on_about
         self.view.exit_callback = self.on_exit
 
+    def open_voxelplan(self, path):
+        """
+        Opens Voxelplan
+        :param path: path to .hed file
+        :return: True if voxelplan is opened, False otherwise
+        """
+        filename, _ = os.path.splitext(path)
+        if not filename:
+            return False
+
+        patient = PatientItem()
+        patient_data = patient.data
+        patient_data.open_ctx(filename + ".ctx")  # Todo catch exceptions
```

Comment:
If `path` is really a path, to the header , then pytrip should accept it when reading ctx file.
There is no need to switch extension to `*.ctx`. pytrip will automatically figure out where ctx file is and if it is gizpped or not.